app/assets/stylesheets/frontend/views/_worldwide-organisations.scss

```diff
@@ -215,3 +141,12 @@
     }
   }
 }
+
+.contact-section {
+  border-top: 1px solid govuk-colour("mid-grey");
+  padding: govuk-spacing(6) 0;
```

Comment:
Ponder: you could use `govuk-!-padding-6` on the class - has me wondering which technique would be desirable first _as a rule_ 🤔

Reply:
I'd rather put it here because it feels neater to me to output this as a shorthand style rule hidden in the CSS than the classes necessary to replicate it (I think `govuk-!-padding-6-bottom` and `govuk-!-padding-6-top`). It also feels more inline with the component concept to isolate the style here.

For what it's worth, the way I treat the helper classes is if I need one specific style or one specific bit of markup like a container div with some bottom margin. The same way might use a `strong` tag for example.